android/modules/calendar/src/java/ti/modules/titanium/calendar/RecurrenceRuleProxy.java

```diff
@@ -126,7 +126,7 @@ public String generateRRULEString()
 							weeklyRecurrencesString.append(",");
 						}
 					}
-					finalRRule.append(weeklyRecurrencesString.toString());
+					finalRRule.append(weeklyRecurrencesString);
```

Comment:
`weeklyRecurrencesString` is a `StringBuilder`, so I'm curious if this parses to a `String` automatically these days

Reply:
https://docs.oracle.com/javase/7/docs/api/java/lang/StringBuilder.html#append(java.lang.StringBuffer)
StringBuilder is a valid append parameter